controllers/product.js

```diff
@@ -0,0 +1,210 @@
+const path = require('path');
+const fs = require('fs');
+const crypto = require('crypto');
+const { EventEmitter } = require('events');
+const { setImagesList, setVideosList } = require('../services/services');
+
+const eventEmitter = new EventEmitter();
+
+const now = new Date();
+const today = now.toUTCString().slice(5, 25);
+
+eventEmitter.on('fileUploadFailed', () => {
```

Comment:
[Append Example](https://nodejs.org/en/learn/manipulating-files/writing-files-with-nodejs#appending-content-to-a-file
)
You can use `appendFile` function to add a new line to the end of the file